### DISCUSSION TITLE
How to know how many times one task have retried in Prefect 2?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [PrefectHQ/prefect#11699] [Q&A]
I hope to know how many times one task have retried in Prefect 2.

I found this solution in Prefect 1 https://github.com/PrefectHQ/prefect/discussions/5193#discussioncomment-1745127

I am wondering how to do it in Prefect? Thanks! 😃

**Comment by tonal** [comment]
see #7576
In **retry_condition_fn**
